design-proposals/enhancements.md

```diff
@@ -0,0 +1,109 @@
+# Overview
+
+Rework the existing KubeVirt design proposal process into a more
+formal - but still lean - and SIG-aligned Enhancement Proposal process.
+
+## Motivation
+
+Today it's a [very small group](https://github.com/kubevirt/community/blob/main/OWNERS#L7-L14)
+which can approve design-proposals.
+While certain individuals can approve design proposals, once they are merged
+it is not clear who is owning them post-merge.
+
+In this proposal we are proposing to change the process and shape them
+around SIGs.
+
+At its core, this proposal requires:
+
+1. designs to be sponsored by a SIG
+2. to make one SIG responsible for the design process (reviews of design, code, documentation)
+3. to make one SIG responsible for managing the design process (collaboration with other SIGs as needed)
+4. to make one SIG to own the feature once it has been merged. The SIG is responsible for maintaining, fixing, running, _everything-it_.
+
+This has a few effects - see the following Goals section.
+
+## Goals
+
+1. The design process now has a mechanism to distribute designs among SIGs
+2. SIG approvers are empowered to approve designs, increase the approvers pool
```

Comment:
Unclear. SIG approvers can increase the approvers pool?
Or, the fact that SIG approvers can approve designs increases the approvers pool? In which case:
```suggestion
2. SIG approvers are empowered to approve designs, increasing the approvers pool
```
But does it really? Technically, most sigs have less approvers than kubevirt/community does today...

Reply:
The second. Because we have sig approvers, we implicitly increase the pool.

Yes, sigs have less approvers then root, but: the sum of sig approvers should venetually be larger than the amount of root approvers.